src/content/docs/en/pages/main-menu/reference/services/support/http-error-status-codes.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: Azion HTTP Error Status Codes
+description: These are the most common web and server errors and how to solve them.
+meta_tags: services, support, documentation, system error
+namespace: documentation_error_status_codes
+permalink: /documentation/services/support/http-error-status-codes/
+---
+
+While using third-party web services implemented and hosted on Azion you may encounter some common web and server errors. These errors can be related to a bad request or missing content, authentication issues, servers malfunctioning, among others.
+
+To inform you of these errors, you'll receive a message similar to this:
+
+![Example - HTTP Error Status Codes](/assets/docs/images/uploads/http-error-status-codes.png)
+
+- **Error Details**: date and time when the error occurred.
+- **Request ID**: a unique identifier for your request. It can be used for further investigation or support.
+- **Status Code**: the HTTP status code that indicates the specific type of error encountered.
+- **POP**: the Azion's edge location that processed your request.
+
+When Azion's system returns an error page, the reason for it could be related to configurations, authorization mechanisms, or a failing connection with the third-party web services you're using. For example, when your bank or e-commerce application is hosted in Azion. To check Azion's platform health status, access [Azion status page](https://status.azion.com "Status page").
```

Comment:
```suggestion
When Azion's system returns an error page, it could be related to configurations, authorization mechanisms, or a failing connection with the third-party web services you're using. For example, when your bank or e-commerce application is hosted in Azion. To check Azion's platform health status, access [Azion status page](https://status.azion.com "Status page").
```